src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php

```diff
@@ -47,6 +47,12 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
      */
     public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null)
     {
+        if (false === $label) {
+            @trigger_error(sprintf('Passing false as $label to %s is deprecated in Symfony 4.4 and will trigger a TypeError in 5.0, pass a callable that returns false instead.', __METHOD__), E_USER_DEPRECATED);
+
+            $label = static function () { return false; };
```

Comment:
if you do this, you could add type declarations to the private methods of the class. and this https://github.com/symfony/symfony/blob/7fc6c00d1524b6060b6821737e8e2477457ff01a/src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php#L146 does not have any value anymore, does it?

Reply:
Agree, done.